compiler/rustc_codegen_llvm/src/coverageinfo/mapgen.rs

```diff
@@ -268,20 +261,17 @@ fn add_unused_functions(cx: &CodegenCx<'_, '_>) {
             && !usage.used_via_inlining.contains(&def_id.to_def_id())
     };
 
-    // Scan for unused functions that were instrumented for coverage.
-    for def_id in tcx.mir_keys(()).iter().copied().filter(|&def_id| is_unused_fn(def_id)) {
-        // Get the coverage info from MIR, skipping functions that were never instrumented.
-        let body = tcx.optimized_mir(def_id);
-        let Some(function_coverage_info) = body.function_coverage_info.as_deref() else { continue };
+    // FIXME(79651): Consider trying to filter out dummy instantiations of
```

Comment:
Nit: `FIXME(#79651)` will make github render the issue number as a clickable link during git blame or rich source view :P

Reply:
This is existing code being reindented, but I might as well fix it anyway.